tests/unit/Trace/Reporter/ZipkinReporterTest.php

```diff
@@ -77,4 +78,30 @@ public function testFormatsTrace()
             }
         }
     }
+
+    public function testSpanKind()
```

Comment:
It is mainly a function of updating environments. The latest zipkin server runs both apis and will do so for a very long time.

Since this is a new library, you can nudge folks to ensure they are running latest. That said, some of the emulators may not expose v2 yet, but it is far easier to code than what they did in v1. There are tradeoffs, but one advantage in a new codebase is you could make a choice to have a single configuration for zipkin vs a knob and both styles. If folks can't upgrade, you could resurrect the v1. Just thoughts.